src/kernels/Conv_Winograd_v30_3_1_gfx10_fp16_dot2_f2x3_dilation2.inc

```diff
@@ -29,71 +29,120 @@
     .long  0xFF000000 + (\vsrc0 + (\q0 << 8) + (\q1 << 10) + (\q2 << 12) + (\q3 << 14))
 .endm
 
+.macro _v_add_f16_e32_1_gfx1x vdst:req, vsrc:req
+    .long  0x640000FF + (\vdst << 17) + (\vsrc << 9)
+    .long  0x00003C00
+.endm
+
+.macro _v_pk_add_f16_gfx1x vdst:req, vsrc0:req, vsrc1:req
+    .long  0xCC0F0000 + (\vdst << 0)
+    .long  0x18020100 + (\vsrc0 << 0) + (\vsrc1 << 9)
+.endm
+
+.macro _v_pk_add_f16_0_gfx1x vdst:req, vsrc0:req
+    .long  0xCC0F0000 + (\vdst << 0)
+    .long  0x00010100 + (\vsrc0 << 0)
+.endm
```

Comment:
This macro implicitly sets `op_sel_hi` to 1 for vsrc0. Please update the macro name accordingly and check other gfx10/gfx11 _.inc_ files for the same macro.

Reply:
Precise naming can be painful. Shall we keep all the encoding flavors in the macro or keep it simple?

There is an example of fury version where it can be harder to maintain:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/8ca1e19db201a86e7f46e8d9d94149eb15b2ba9d/src/kernels/Conv_Winograd_Fury_v0_0_1_gfx11_fp16_fp16acc_f2x3_stride1.inc#L48